pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -159,6 +160,9 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
   /** The current config state. */
   protected cfg: BackendSettings | undefined;
 
+  /** Indicates wherther the current installation is a privileged install or not. */
+  protected privilegedServiceEnabled: boolean;
```

Comment:
FWIW, this can instead be:
```suggestion
  protected privilegedServiceEnabled = false;
```
And it wouldn't need to be in the constructor.